lib/SILGen/SILGenPattern.cpp

```diff
@@ -131,7 +131,7 @@ static bool isDirectlyRefutablePattern(const Pattern *p) {
   case PatternKind::Typed:
   case PatternKind::Var:
     return isDirectlyRefutablePattern(p->getSemanticsProvidingPattern());
-  }  
+  }
```

Comment:
Please do this in a separate PR thing. It distracts from the review.

Reply:
Sorry, I will remove it.